resources/app/login/Login.tsx

```diff
@@ -15,91 +15,101 @@ import {
+            flexDirection="column"
+            justifyContent="center"
+            alignItems="center"
+            width="100wh"
+            height="100vh"
         >
             <Helmet>
-                <title>{document.title}</title>
+                <title>{document.title} LOGIN</title>
                 <meta name="description" content="This is a description" />
             </Helmet>
             <Stack
-            flexDir="column"
-            mb="2"
-            justifyContent="center"
-            alignItems="center"
-            >
-            <Box
-            minW={{base:"90%",md:"300px"}}
-            backgroundColor={useColorModeValue("white", "backgroundDark")}
-            borderRadius={8}
+                flexDir="column"
+                mb="2"
+                justifyContent="center"
+                alignItems="center"
             >
-                <form onSubmit={handleSubmit}>
-                    <Stack
-                    spacing="4"
-                    p="1rem"
-                    boxShadow="md"
-                    >
-                        <FormControl
-                        size="sm"
-                        >
-                            <FormLabel htmlFor="email">Email Address</FormLabel>
-                            <Input type="email" name="email" placeholder="Email" value={getUsername} onChange={(e:any) => setUsername(e.target.value)} required />
-                        </FormControl>
-                        <FormControl
-                        size="sm"
-                        >
-                            <FormLabel htmlFor="password">Password</FormLabel>
-                            <Input type="password" name="password" placeholder="Password" value={getPassword} onChange={(e:any) => setPassword(e.target.value)} required />
-
-                        </FormControl>
-                        <Button 
-                        type="submit"
-                        colorScheme="teal"
-                        size="sm"
-                        variant="solid"
-                        >LOGIN</Button>
-                        <HStack
-                        flexDir="row"
-                        justifyContent="center"
-                        alignItems="center"
-                        borderTop="1px"
-                        borderColor={useColorModeValue("gray.200", "gray.800")}
-                        >
-                            <Link to="/db/manager" style={{paddingTop:"1rem"}}>
-                                <Text fontSize={11.5} color="teal">Database Manager</Text>
-                            </Link>
-                        </HStack>
-                    </Stack>    
-                </form>
-
-            </Box>
-                
+                <Box
+                    minW={{ base: "90%", md: "400px" }}
+                    backgroundColor={useColorModeValue("white", "dark.card")}
+                    borderRadius={8}
+                >
+                    <LoginForm />
```

Comment:
Codacy has a fix for the issue: [Replace `······<LoginForm` with `<LoginForm·`](https://app.codacy.com/gh/bluespada/go-telescope/pullRequest?prid=9653145)
```suggestion
          <LoginForm />
```